.github/actions/make_init/action.yml

```diff
@@ -0,0 +1,77 @@
+name: "Initialize Virtual Env"
+runs:
+  using: composite
+  steps:
+    - name: Restore nvm, pre-commit cache, & node modules from cache
+      id: cache-nvm-node-modules
+      uses: actions/cache@v3
+      with:
+        path: |
+          ~/.nvm
+          ~/.cache
+          frontend/node_modules
+        key: v13-nvm-node-modules-${{ hashFiles('**/yarn.lock.md5') }}-${{ hashFiles('**/.pre-commit-config.yaml') }}
+    - name: Install nvm, pre-commit cache and node_modules
+      if: steps.cache-nvm-node-modules.cache-hit != 'true'
+      run: |
+        if [ ! -d ~/.nvm ] ; then
+          cd ..
+          export NVM_DIR="$HOME/.nvm"
+          [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+          mkdir $NVM_DIR
+          curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+          echo "source $NVM_DIR/nvm.sh --install" >> "$HOME/.bash_profile"
+          cd -
+          source $HOME/.bash_profile
+          echo "lts/*" > .nvmrc
```

Comment:
We already have `.nvmrc` file. Do we have to overwrite it? I'm also not sure if using nvm on Github Action is the best idea.

Reply:
+1 to just using the available GitHub action for node installation rather than doing it ourselves via `nvm`. At the very least, we can drop overwriting `.nvmrc` here since it already exists in the repo.

I believe in an open source meeting, there was some hesitancy around depending on an external action to install node since it adds another dependency when it only takes a few lines of code to do it ourselves, but I'm pretty skeptical of an action as simple as one that installs node breaking frequently, and even then we can just pin it to a version so that isn't actually a concern.

Reply:
Resolved via implementation of github action setup-node 👍🏼